HdmiCec/HdmiCec.cpp

```diff
@@ -322,7 +322,7 @@ namespace WPEFramework
             cecSetting.IElement::ToFile(file);
 
             file.Close();
-
+            Utils::syncPersistFile(CEC_SETTING_ENABLED_FILE);
```

Comment:
Pls move the entire PersistSettings to Utils and name it something like PersistJsonSetting(file, Key, Val). Also don't need to create separate func for syncing. can be done within PersistJsonSetting.